demo/server.js

```diff
@@ -150,4 +150,52 @@ initSqlJs().then(SQL => {
   });
 
   app.listen(port, () => console.log(`Listening on port ${port}`));
+
+   // ACCURACY RECORD MANAGEMENT
+   function presistAccuracyRecord(accuracy) {
+    db.run('INSERT INTO accuracy VALUES (?, ?, ?, ?, CURRENT_TIMESTAMP);', [
+      accuracy.transactionHash,
+      accuracy.blockNumber,
+      accuracy.modelId,
+      accuracy.accuracy,
+    ]);
+    fs.writeFile(dbPath, Buffer.from(db.export()), () => {});
+  }
+  
+ // Add a new accuracy record for a model
+  app.post('/api/accuracy', jsonParser, (req, res) => {
+    const body = req.body
+    presistAccuracyRecord(body);
+    return res.sendStatus(200);
+  });
+
+  // Get the accuracy history 
+  app.get('/api/accuracy/model', (req, res) => {
+    const { modelId  } = req.query;
+    if (modelId != null) {
+      const getStmt = db.prepare('SELECT * FROM accuracy where model_id == $modelId ORDER BY timestamp;',
+      {
+        $modelId: modelId
+      })
+      const accuracyHistory = [];
+      while (getStmt.step()) {
+        const accuracy = getStmt.get();
+        accuracyHistory.push({
+          transactionHash: accuracy[0],
+          blockNumber: accuracy[1],
+          modelId: accuracy[2],
+          accuracy: accuracy[3],
+          timestamp: accuracy[4],
+        });
+      }
+      getStmt.free();
+      if (accuracyHistory.length) {
+        res.send({ accuracyHistory });
+      } else {
+        res.send({ message: "Not Valid modelId" });
+      }
+    } else {
+      return res.status(400).send({ message: "Null" });
```

Comment:
The message should be clearer about what the issue was.

Reply:
okay i try to improve the messages